recipes/spylon/meta.yaml

```diff
@@ -25,6 +25,7 @@ requirements:
     - html5lib
     - lxml
     - beautifulsoup4
+    - ca-certificates
```

Comment:
Yeah originally as part of my setup.py I fetched some things from https://spark.apache.org and that had ssl cert failures when fetching via urllib.
I altered the package to instead distribute them myself.